### DISCUSSION TITLE
Quick way to check if there are non-header rows or not

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2069] [Q&A] [answered]
Is there a quick way to specifically check if a file has non-header rows or not? For example:

```csv
a,b
```

We assume the first row is a header row and there are no non-header rows so some value like true or false is returned based on implementation.

**Comment by jqnatividad** [comment, marked as answer]
Hhmmm... for a single row, it's not possible.

But if we have at least a few rows, `sniff` makes a good guesstimate, but it still sometimes fails for otherwise valid CSVs.
